src/renderer/components/ft-subscribe-button/ft-subscribe-button.css

```diff
@@ -1,6 +1,7 @@
-.subscribeButton {
+.subscribeButton.btn {
```

Comment:
```suggestion
/* Ensures style here overrides style of .btn */
.subscribeButton.btn {
```

Reply:
Even though this is a pretty obvious/borderline case, I can see exactly why you're suggesting this, and I need to be more conscientious of this. Adding it in.

Reply:
Project specific stuff (which cannot be searched via doc) should be explained when possible
(like the `.btn` style got higher priority than `.subscribeButton` here)

We spend more time reading code than writing code :)